Package.swift

```diff
@@ -0,0 +1,213 @@
+            dependencies: ["IceCpp"],
+            path: "cpp",
+            exclude: [
+                "test",
+                "src/IceDiscovery/build",
+                "src/IceDiscovery/msbuild",
+                "src/IceDiscovery/IceDiscovery.rc",
+                "src/IceDiscovery/Makefile.mk",
+                "src/IceDiscovery/generated/IceDiscovery.ice.d",
+            ],
+            sources: ["src/IceDiscovery"],
+            publicHeadersPath: "src/IceDiscovery/generated",
+            cxxSettings: iceCppSettings
+        ),
+        .target(
+            name: "IceLocatorDiscoveryCpp",
+            dependencies: ["IceCpp"],
+            path: "cpp",
+            exclude: [
+                "test",
+                "src/IceLocatorDiscovery/build",
+                "src/IceLocatorDiscovery/msbuild",
+                "src/IceLocatorDiscovery/IceLocatorDiscovery.rc",
+                "src/IceLocatorDiscovery/Makefile.mk",
+                "src/IceLocatorDiscovery/generated/IceLocatorDiscovery.ice.d",
+            ],
+            sources: ["src/IceLocatorDiscovery"],
+            publicHeadersPath: "src/IceLocatorDiscovery/generated",
+            cxxSettings: iceCppSettings
+        ),
+
+        .target(
+            name: "IceUtilCpp",
+            path: "cpp",
+            exclude: [
+                "test",
+                "src/IceUtil/build",
+                "src/IceUtil/msbuild",
+                "src/IceUtil/Makefile.mk",
+            ],
+            sources: iceUtilSources,
+            publicHeadersPath: "src/IceUtil",  // dummy path as we can't re-use include/Ice
+            cxxSettings: iceCppSettings
+        ),
+        .target(
+            name: "SliceCpp",
+            dependencies: ["IceUtilCpp", "mcpp"],
+            path: "cpp",
+            exclude: [
+                "test",
+                "src/Slice/build",
+                "src/Slice/msbuild",
+                "src/Slice/Scanner.l",
+                "src/Slice/Grammar.y",
+                "src/Slice/Makefile.mk",
+            ],
+            sources: ["src/Slice"],
+            publicHeadersPath: "src/Slice",
+            cxxSettings: iceCppSettings
+        ),
+        .executableTarget(
+            name: "slice2swift",
+            dependencies: ["SliceCpp", "IceUtilCpp"],
+            path: "cpp",
+            exclude: [
+                "test",
+                "src/slice2swift/build",
+                "src/slice2swift/msbuild",
+                "src/slice2swift/Slice2Swift.rc",
+            ],
+            sources: ["src/slice2swift"],
+            publicHeadersPath: "src/slice2swift",
+            cxxSettings: iceCppSettings
+        ),
+        .plugin(
+            name: "CompileSlice",
+            capability: .buildTool(),
+            dependencies: ["slice2swift"],
+            path: "swift/src/CompileSlicePlugin"
+        ),
+    ],
+    swiftLanguageVersions: [SwiftVersion.v5],
+    cxxLanguageStandard: .cxx17
```

Comment:
Can we use .cxx20 like in C++ macOS builds?

Reply:
I think so!